integration/fuse/bin/alluxio-fuse

```diff
@@ -40,7 +40,7 @@ mount_fuse() {
     full_mount_option="big_writes"
   fi
 
-  if [[ ${NO_DAEMON} -eq "1" ]]; then
+  if [[ ${NO_DAEMON} = true ]]; then
```

Comment:
Double equal is preferred when comparing strings
```
For clarity, use == for equality rather than = even though both work. The former encourages the use of [[ and the latter can be confused with an assignment. 
```